app/models/ext_management_system.rb

```diff
@@ -578,6 +578,11 @@ def stop_event_monitor
   end
 
   def stop_event_monitor_queue
+    unless id
```

Comment:
ah, I see, I don't have these checks in openstack/manager_mixin, which is overwriting those methods. Ok, let me fix that